docs/source/user/getting-started.rst

```diff
@@ -4,32 +4,149 @@ Getting Started
 .. contents:: |toctitle|
     :local:
 
+Goal
+----
+
+This section will walk you through the process of getting started with OpenDP. By the end, you should have a working OpenDP installation, and you'll be ready to explore OpenDP programming.
+
 Installation
 ------------
 
-The first step in using OpenDP is to install the OpenDP library.
+OpenDP is built with a layered architecture. The core consists of a native library, implemented in Rust. On top of this core are bindings for using OpenDP from other languages (currently only Python, but we hope to add bindings for more languages in the future).
+
+In order to use OpenDP in your applications, the first step is installing the library so that it's accessible to your local environment. Depending on how you intend to use OpenDP, these steps may vary.
+
+Platforms
+^^^^^^^^^
+
+OpenDP is built for the following platforms:
+
+* Python 3.6-3.9
+* Linux, x86 64-bit, versions compatible with `manylinux <https://github.com/pypa/manylinux>`_ (others may work)
+* macOS, x86 64-bit, version 10.13 or later
+* Windows, x86 64-bit, version 7 or later
+
+Other platforms may work, but will require `building from source <#building-opendp-from-source>`_.
+
+Installing OpenDP for Python from PyPI
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+OpenDP is distributed as a `Python package on PyPI <https://pypi.org/project/opendp/>`_. You can install it using ``pip`` or another package management tool:
+
+.. prompt:: bash
+
+    pip install opendp
+
+.. note::
+
+    The OpenDP Python package contains binaries for all supported platforms, so you don't need to worry about specifying a specific platform.
+
+OpenDP has no dependencies, so at this point, you should be good to go! You can confirm your installation in Python by importing the top-level module ``opendp``:
```

Comment:
This is such a strange and wonderful thing to say. I know you're proud of it. But pip automatically installs dependencies anyways.

Reply:
Spoilsport!